script2.js

```diff
@@ -0,0 +1,31 @@
+let emotions = {
+    upset: 0,
+    unhappy: 0,
+    neutral: 0,
+    happy: 0,
+    joyful: 0,
+    total: 0
+  };
+  
+  function submitSentiment(event) {
```

Comment:
Nice work on the JS! Could you modify the form handling to use an eventListener for the form submit button (I believe this is how it's typically done)